open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+found at <https://www.npmjs.com/policies/personal-terms>.
+
+npm Open Source and any Paid Services you may agree to use are together
+called _npm Services_ throughout these Terms.
+
+## Legal Agreement
+
+You may only access or use npm Services by agreeing to these Terms. If
+npm adds any additional functionality to npm Services, you must agree
+to these Terms to use new features, too. You show your agreement with
+npm on these Terms by clicking a check box in the process of creating an
+user account on the Website (your _Account_) or by accessing or using
+npm Services without creating an account. The agreement between you and
+npm is a legally binding contract (this _Agreement_).
+
+## Changes
+
+npm may change these Terms and the additional terms for Paid Services
+in the future. npm will post changes on the Website with a new "last
+updated" date. If you have an Account, npm will notify you of changes
+by e-mail to the address provided for your Account, by a message on the
+Website, or both. If you do not have an account, npm may notify you of
+changes by a general announcement via the Website, but it is up to you
+to check for changes to these Terms. After receiving notice of changes
+to these Terms, you must accept those changes to continue using npm
+Services. You accept changes to these Terms by continuing to use npm
+Services. npm may change, suspend, or discontinue npm Services at any
+time without notice or liability to you.
+
+## npm Policies
+
+npm respects your privacy and limits use and sharing of information
+about you collected by npm Services. The privacy policy at
+<https://www.npmjs.com/policies/privacy> (the _Privacy Policy_)
+describes these policies. npm will abide by the Privacy Policy and honor
+the privacy settings that you choose via npm Services.
+
+npm respects the exclusive rights of copyright holders and responds
+to notifications about alleged infringement via npm Services per
+the copyright policy at <https://www.npmjs.com/policies/dmca> (the
+_Copyright Policy_).
+
+npm resolves disputes about package names in the Public Registry per the
+policy at <https://www.npmjs.com/policies/disputes> (_Dispute Policy_).
+
+Use of all npm Services is governed by the code of conduct at
+<https://www.npmjs.com/policies/conduct> (_Code of Conduct_).
+
+npm permits use of npm trademarks per the policy at
+<https://www.npmjs.com/policies/trademark>.
+
+## Use of npm Open Source
+
+Subject to these Terms, npm grants you permission to use npm Open
+Source. That permission is not exclusive to you, and you cannot transfer
+it to anyone else.
+
+Your permission to use npm Open Source entitles you to do the following:
+
+1.  You may search for, download, publish, and manage packages of
+    computer code (_Packages_) in the Public Registry, and otherwise
+    interact with the Public Registry, via the command-line tool
+    published by npm at <https://www.npmjs.com/npm/npm> (the _CLI_).
```

Comment:
Incorrect url.  Maybe that should be https://github.com/npm/npm?  or https://www.npmjs.com/package/npm?